src/Workspaces/Core/Portable/Rename/ConflictEngine/Annotations/RenameActionAnnotation.cs

```diff
@@ -23,17 +23,18 @@ internal class RenameActionAnnotation : RenameAnnotation
         public readonly bool IsRenameLocation;
 
         /// <summary>
-        /// A flag indicating if this identifier represents an accessor. E.g. get_Foo (of property Foo).
-        /// </summary>
-        public readonly bool IsAccessorLocation;
-
-        /// <summary>
         /// A flag indicating whether the token at this location has the same ValueText then the original name 
         /// of the symbol that gets renamed.
         /// </summary>
         public readonly bool IsOriginalTextLocation;
 
         /// <summary>
+        /// When replacing the annotated token this string will be prepended to the token's value. This is used when renaming compiler 
+        /// generated fields and methods backing properties (e.g. "get_X" or "_X" for property "X").
+        /// </summary>
+        public readonly string Prefix;
```

Comment:
This comment talks about renaming the backing method, but I don't see any tests for that.